packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -310,6 +315,11 @@ export interface IContextualMenuItem {
   onMouseDown?: (item: IContextualMenuItem, event: any) => void;
 
   /**
+   * The position in the set of this menu item
+   */
+  ariaPosInSet?: number;
```

Comment:
A few more details here would be nice, right now the comment isn't providing much value that couldn't be inferred from the name.